projects/github-actions/repo-gardening/src/tasks/triage-new-issues/index.js

```diff
@@ -14,8 +32,8 @@ const getLabels = require( '../../utils/get-labels' );
  */
 async function hasPriorityLabels( octokit, owner, repo, number ) {
 	const labels = await getLabels( octokit, owner, repo, number );
-	// We're only interested in priority labels.
-	return !! labels.find( label => label.match( /^\[Pri\].*$/ ) );
+	// We're only interested in priority labels, but not if the label is [Pri] TBD.
+	return !! labels.find( label => label.match( /^(?!\[Pri\] TBD)\[Pri\].*$/ ) );
```

Comment:
Might be clearer to just do
```suggestion
	return !! labels.find( label => label !== '[Pri] TBD' && label.match( /^\[Pri\].*$/ ) );
```